src/ui-pages/join-without-merging.tsx

```diff
@@ -34,7 +34,7 @@ export const JoinWithoutMerging = (props: JoinWithoutMergingProps) => {
         </button>
         <button
           disabled={!joinShareId || (!personalDataLabel && !lastPersonalDataLabel)}
-          onClick={joinShare}>
+          onClick={() => joinShare()}>
```

Comment:
Would it be a little better to not use the anonymous function and keep this as `onClick={joinShare}`?

In a [previous change](https://github.com/concord-consortium/codap-shared-table-plugin/pull/101/files), it looks like you added the anonymous function so you could pass an argument. But you're no longer passing an argument here, so maybe it's not necessary?

Reply:
I was quite surprised, but the change from `() => joinShare()` to `joinShare` is what caused the bug. I now realize that when it's just `joinShare`, the click event gets passed to `joinShare` as an argument, which the function definitely wasn't prepared to handle.

Reply:
I see. Good to know.